x/incentives/keeper/distribute.go

```diff
@@ -344,32 +373,80 @@ func (k Keeper) Distribute(ctx sdk.Context, gauges []types.Gauge) (sdk.Coins, er
 
 	locksByDenomCache := make(map[string][]lockuptypes.PeriodLock)
 	totalDistributedCoins := sdk.Coins{}
+	var gaugeDistributedCoins, gaugeDistributedCoinsCL sdk.Coins
```

Comment:
I don't understand why we need to differentiate between these. Can we keep `totalDistributedCoins` only, please?

Reply:
i mainly added it so that we can only return the amount distributed in CL, but fine to remove it rn since we're not doing anything with that value